web/main.js

```diff
@@ -42,7 +45,7 @@ function importFile(f)
 return {
   init : function(canvas)
   {
-    Module._initialize_GL(1040, 1040);
+    Main.setMode('B');
```

Comment:
The new default. I introduced mode B and the autodetect decoder behavior a year ago+, but I didn't want to change the default before giving people time to upgrade.

(I may not push this to cimbar.org right away, but it'll be soon)